### ISSUE TITLE
Prescribe types in columns for `arangoimp`

### ISSUE THREAD

**Issue by neunhoef** [arangodb/arangodb#7563]
Sometimes it is necessary to say that the entries in a certain column of the import should have a certain type like a string. This is in particular necessary for foreign key columns which might contain numerical values, since they would be changed to numbers by the importer by default.

**Comment by jsteemann** [comment]
Note: that above makes sense when importing CSV or TSV files, which do not carry any type information.
When importing JSON/JSONL files the problem does not exist, because values are typed already. However, potentially one wants to convert some attribute values anyway even with JSON.

**Comment by Simran-B** [comment]
I agree that such an option would be helpful, especially for importing ID columns from CSV and use it as _key. Then you don't have to change the source file to fix import issues:
https://groups.google.com/d/msg/arangodb/Fy8aiUsMMrM/X7GaJgdeAQAJ

Should source values that are mapped to `_key` field automatically cast to string even?
